lib/uniffi-zcash/tests/kotlin/orchard.kts

```diff
@@ -22,12 +22,28 @@ fun testSpendingKeyArrayMismatch() {
 }
 testSpendingKeyArrayMismatch()
 
+
```

Comment:
this approach contains the whole csv and gets the method. useful for integration tests I guess